apps/api/src/app/health/health.controller.ts

```diff
@@ -1,28 +1,15 @@
 import { Controller, Get } from '@nestjs/common';
-import { HealthCheck, HealthCheckService, HttpHealthIndicator } from '@nestjs/terminus';
-import { DalService } from '@novu/dal';
+import { HealthCheck, HealthCheckService } from '@nestjs/terminus';
 import { version } from '../../../package.json';
 
 @Controller('health-check')
 export class HealthController {
-  constructor(
-    private healthCheckService: HealthCheckService,
-    private healthIndicator: HttpHealthIndicator,
-    private dalService: DalService
-  ) {}
+  constructor(private healthCheckService: HealthCheckService) {}
 
   @Get()
   @HealthCheck()
-  healthCheck() {
-    return this.healthCheckService.check([
-      async () => {
-        return {
-          db: {
-            status: this.dalService.connection.readyState === 1 ? 'up' : 'down',
-          },
-        };
-      },
```

Comment:
A small question, why did you removed the DAL healthcheck? This one should still be here :)
I only removed it from WS service because it's unused there, but in API it's important we know that the connection to the DB is live.

Reply:
ok